ddev/tests/conftest.py

```diff
@@ -94,6 +95,16 @@ def valid_integrations(local_repo) -> list[str]:
     return [path.name for path in repo.integrations.iter_all(['all'])]
 
 
+@pytest.fixture
+def github_manager(local_repo, config_file, terminal) -> GitHubManager:
```

Comment:
I don't understand what you mean, this is our GitHub client. What do you mean exactly?

Reply:
I thought we already had a `GitHubManager ` fixture somewhere. I thought you were defining a separate one to avoid using the real repo as a fixture, something we also already discussed.

Reply:
ah no I just extracted it from all the tests that are in the test file I'm modifying to avoid copy/paste it

Reply:
ok, gotcha. resolving thread.